constants/__init__.py

```diff
@@ -53,6 +53,15 @@
 CORTEX_MIN_SCORE = 0.85
 # Minimum stake to get data from a cortex validator.
 CORTEX_MIN_STAKE = 100_000
+# The uid for the Prompting subnet.
+PROMPTING_SUBNET_UID = 1
+# The Prompting validator WANDB project and filters
+PROMPTING_WANDB_PROJECT = "macrocosmos/prompting-validators"
+PROMPTING_MAX_AGE = dt.timedelta(hours=4)
+# Percentage of promping miners who must have gotten the question correct to include in the eval set.
+PROMPTING_MIN_CORRECT_MINERS = 0
+# Minimum stake to get data from a cortex validator.
+PROMPTING_MIN_STAKE = 100_000
```

Comment:
Optional: It seems unlikely we'll use separate min stakes for 18 vs 1. Could probably just have 1 value for it.

If keeping (which I'm totally fine with), the comment says "cortext validator" instead of "prompting validator"

Reply:
Reduced to one constant value.